python/ext/third_party/mc/cMesher.cpp

```diff
@@ -92,8 +92,9 @@ bool cMesher::write_obj(const unsigned int id, const std::string &filename) {
 }
 
 meshobj cMesher::get_mesh(const unsigned int id, const bool generate_normals, const int simplification_factor, const int max_simplification_error) {
-    meshobj obj;
+  meshobj obj;
 
+  if (mc.count(id) > 0) { // MC produces no triangles if either none or all voxels were labeled!
```

Comment:
One small style change I'd prefer. Would you mind making it instead
```
if (mc.cound(id) == 0) {
    return obj;
}
```

I find guards that protect large blocks of code introduce a bit of mental overhead so I like to avoid them when possible.